2003-02-06  Jakub Jelinek  <jakub@redhat.com>

	* elf64-alpha.c (elf64_alpha_relax_section): Don't crash if
	local_got_entries is NULL.

--- bfd/elf64-alpha.c.jj	2003-02-06 10:53:56.000000000 -0500
+++ bfd/elf64-alpha.c	2003-02-06 10:53:56.000000000 -0500
@@ -2184,7 +2184,13 @@ elf64_alpha_relax_section (abfd, sec, li
 
 	  info.h = NULL;
 	  info.other = isym->st_other;
-	  info.first_gotent = &local_got_entries[r_symndx];
+	  if (local_got_entries)
+	    info.first_gotent = &local_got_entries[r_symndx];
+	  else
+	    {
+	      info.first_gotent = &info.gotent;
+	      info.gotent = NULL;
+	    }
 	}
       else
 	{

